tools/mason/MasonEnv.chpl

```diff
@@ -112,6 +111,14 @@ proc MASON_REGISTRY {
   return registries;
 }
 
+proc getSpackRoot() : string {
```

Comment:
What about the name `SPACK_ROOT`? It's consistent with the `MASON_HOME`, `MASON_REGISTRY` precedures.

Reply:
@Rapiz1  Thanks for the comment, turns out we don't need a function at all.